### ISSUE TITLE
Please make a new release that includes support for specifying reader schema

### ISSUE THREAD

**Issue by andershammar** [databricks/spark-avro#177]
The following two commits did not make it into the 3.0.1 release. They are very important and highly anticipated.

```
b078cca fix #96, support user-define schema when reading avro files
7f0336c Support user defined avro json format schema
```

Can you please make a new release that includes this functionality?

Thanks!
/Anders

**Comment by JoshRosen** [comment]
I intentionally left these two commits out of the 3.0.1 release, which was intended to be a minimal [patch release](http://semver.org/) in order to include two critical bugfixes. New features, on the other hand, can only be added in a minor release, so the next release to include those fixes will be 3.1.0.

Do you know whether the added features have been adequately QA'd / tested? I'm wondering whether we might also want to pull in fixes / enhancements like #173 before cutting the release.

**Comment by JoshRosen** [comment]
I'm going to try to get a new release out this week, but will try to close out the open PRs and issues first in order to make sure that nothing major misses this release.

**Comment by JoshRosen** [comment]
Just released this in 3.1.0: https://github.com/databricks/spark-avro/releases/tag/v3.1.0
